src/hooks/protections/can_apply_data.cpp

```diff
@@ -1318,6 +1318,16 @@ namespace big
 					notify::crash_blocked(sender, "invalid object model");
 					return true;
 				}
+				if (protection::is_cage_object(creation_node->m_model) && g.protections.cage_protection)
+				{ // This will trigger for any player caging any player, Due to a issue with m_dummy_position data
+					auto plyr = g_player_service->get_by_id(sender->m_player_id);
```

Comment:
`sender_plyr` variable is there already.

Reply:
Will fix it later. Also need to do something about the protections tab since it looks very weird now i have added 2 new ones.

Reply:
yes the checkboxes need to fit well

Reply:
Yes, It looks weird will have to fix it in a bit lol but we still need to find what event was removed before this can be merged